runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java

```diff
@@ -205,15 +205,15 @@ public String create(PipelineOptions options) {
   void setFilesToStage(List<String> value);
 
   /**
-   * Specifies what type of persistent disk should be used. The value should be a full or partial
-   * URL of a disk type resource, e.g., zones/us-central1-f/disks/pd-standard. For
-   * more information, see the
-   * <a href="https://cloud.google.com/compute/docs/reference/latest/diskTypes">API reference
-   * documentation for DiskTypes</a>.
+   * Specifies what type of persistent disk is used. The value is a full disk type resource,
+   * e.g., compute.googleapis.com/projects//zones//diskTypes/pd-ssd. For more information,
```

Comment:
The website uses https://www.googleapis.com/compute/v1/projects/myproject/zones/zone/diskTypes/pd-ssd as an example which differs from the example that your provide since it contains a scheme and also a different host

Why the discrepancy?